custom-widget-builder/src/CustomWidgetBuilder.ts

```diff
@@ -74,11 +73,12 @@ export class CustomWidgetBuilder {
     //     │       ├── MyInput.js
     //     │       └── myInput.tpl.runtime.html
     //     ├── myInput.tpl.html
-    //     └── widgetWc.json
+    //     └── myInput.json
 
     let propInfo = CustomWidgetBuilder.getPropertiesFromFile(propertiesFile);
-    // Add assets to the json properties file
-    propInfo.assets = CustomWidgetBuilder.getAssets(propInfo.id, wcBundle);
+    // Add bundles to the json properties file
+    propInfo.jsBundle =  `resources/assets/js/${path.basename(wcBundle)}`;
```

Comment:
We don't need resources here.
`Resources` folder is only here in zip. When file is import/upload, the path is `widgetId/assets/js`.

Reply:
Ye I agree. Since json properties file being already in the 'resources' folder we don't need it.